packages/ddp-server/livedata_server.js

```diff
@@ -1,5 +1,7 @@
 DDPServer = {};
 
+import { DiffSequence} from "../diff-sequence/diff"
```

Comment:
Since this is a different package, you will need to import it like you would in an app. Same in `packages/ddp-client/common/livedata_connection.js`. Also double check that the dependency is in `package.js`

Reply:
Will do. Wrapping up these changes and final comments on the code. I will change it from a draft once ready for review.

Thanks!